src/photos.cs

```diff
@@ -904,6 +948,7 @@ interface PHImageKeys {
 	delegate void PHImageManagerRequestAvAssetHandler (AVAsset asset, AVAudioMix audioMix, NSDictionary info);
 #endif
 	delegate void PHImageManagerRequestLivePhoto (PHLivePhoto livePhoto, NSDictionary info);
+	delegate void PHImageManagerRequestImageDataHandler ([NullAllowed] NSData imageData, [NullAllowed] string dataUTI, CGImagePropertyOrientation orientation, [NullAllowed] NSDictionary info);
```

Comment:
ImageUTI to imageUti, AFAIK we use CamelCase for args, like sourceUrl

Reply:
Good Catch!